### PR TITLE
Routing form UI update

### LINKED ISSUE #7413: Routing form UI update
Current design

Screenshot 2023-02-26 at 5 17 55 PM

Should be

Screenshot 2023-02-26 at 5 17 06 PM

https://www.figma.com/file/xk4HOxtSI82J0F7enMxeak/Cal---Live?node-id=26%3A101718

### PR DESCRIPTION
**Pull request by gitstart-calcom**
## What does this PR do?

Fixes #7413 

## Demo
https://user-images.githubusercontent.com/26940294/222206271-1bc38d00-0870-4eb7-b2f2-88e72726e10c.mov

**Environment**: Staging(main branch) / Production

### CODE REVIEW

File: packages/features/shell/Shell.tsx [hunk 1]
```diff
@@ -782,8 +782,8 @@ export function ShellMain(props: LayoutProps) {
     <>
       <div
         className={classNames(
-          "flex items-center md:mt-0 md:mb-6",
-          props.smallHeading ? "lg:mb-7" : "lg:mb-8"
+          "flex items-center md:mt-0 md:mb-8",
+          props.smallHeading ? "lg:mb-9" : "lg:mb-10"
```
Comment (CarinaWolli) [inline packages/features/shell/Shell.tsx:786]: Looks like routing forms has a larger gap between shell and the list than for example`/event-types` or `/bookings`  ([Figma Design](https://www.figma.com/file/xk4HOxtSI82J0F7enMxeak/Cal---Live?node-id=20%3A15579&t=yqnXxCFaOm5sFRs0-0))

So let's not make the changes in the shell itself and instead add it directly to routing forms. @Jaibles Or should we make the gap below the shell the same as it is for event types? I think workflows has a larger gap too

**Review by CarinaWolli — CHANGES_REQUESTED** [review]
Just one small comment everything else looks good 🙏

File: packages/features/shell/Shell.tsx (hunk 1 above)
Comment (gitstart-calcom) [reply packages/features/shell/Shell.tsx:786]: What should we do @Jaibles?

Comment (Jaibles) [reply packages/features/shell/Shell.tsx:786]: There should be a space of 32px.
CleanShot 2023-03-06 at 17 54 03@2x

CleanShot 2023-03-06 at 17 53 51@2x

Comment (CarinaWolli) [reply packages/features/shell/Shell.tsx:786]: Thank you! @gitstart means we should kee `mb` as it is. Once this is fixed I can approve 🙏

**Comment by gitstart-calcom** [comment]
Hi @CarinaWolli, requested changes have been made. Please take a look

**Review by CarinaWolli — APPROVED** [review]
Thank you 🙏

**Review by PeerRich — APPROVED** [review]

**Pull request merged**